apps/dashboard/app/views/files/index.html.erb

```diff
@@ -6,6 +6,7 @@
     <%= render partial: 'shell_dropdown' %>
   <% end %>
 
+    <button id="refresh-btn" type="button" class="btn btn-outline-dark btn-sm"><i class="fa fa-rotate-right" aria-hidden="true"></i> Refresh</button>
```

Comment:
This content comes from these rules, so looks like there is no `fas` modifier in this version of FA.

![image](https://user-images.githubusercontent.com/4874123/181317827-7b1ae13d-5723-41b0-a7c0-e9f73727ec92.png)